server/views/pages/create/checkAnswers.njk

```diff
@@ -33,8 +43,11 @@
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
-
-            {{ inductionMeetingDetails(licence, isInProgress) }}
+            {% if not licence.isVariation %}
```

Comment:
Mixture of `if not isVariation` and `if isVariation` - might be cleaner with consistency?